src/plone/restapi/services/system/configure.zcml

```diff
@@ -0,0 +1,13 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:plone="http://namespaces.plone.org/plone">
+
+  <plone:service
+    method="GET"
+    for="zope.interface.Interface"
+    factory=".get.SystemGet"
+    name="@system"
+    permission="zope2.View"
```

Comment:
I am sure we do not want to expose this on view permission level. Use same permission as the view, see https://github.com/plone/Products.CMFPlone/blob/d1fcfd07972b65a8aeeb6a2b9f0f0f0da378c310/Products/CMFPlone/controlpanel/browser/configure.zcml#L26

Reply:
Fixed